src/UIKit/UIEnums.cs

```diff
@@ -779,6 +779,10 @@ public enum UIDataDetectorType : ulong {
 		FlightNumber           = 1 << 5,
 		[iOS (10,0)]
 		LookupSuggestion       = 1 << 6,
+		[NoWatch, NoTV, iOS (16,0), MacCatalyst (16,0)]
+		Money = 1uL << 7,
```

Comment:
minor - either use `1 << 7` here to be consistent with all the others or change the others to be `1uL << n`. I would personally do the former since the backing type is `ulong` so the `uL` is redundant.